projects/batfish-common-protocol/src/main/java/org/batfish/common/Container.java

```diff
@@ -5,36 +5,35 @@
 import com.google.common.base.MoreObjects;
 import java.nio.file.Path;
 import java.util.Objects;
-import java.util.SortedSet;
 
 public final class Container {
   private static final String NAME_VAR = "name";
-  private static final String TESTRIGS_VAR = "testrigs";
+  private static final String TESTRIGS_URI_VAR = "testrigsUri";
   private static final String ANALYSIS_VAR = "analysis";
 
   private String _name;
-  private SortedSet<String> _testrigs;
+  private String _testrigsUri;
   private Path _analysis;
 
   @JsonCreator
-  public static Container of(
-      @JsonProperty(NAME_VAR) String name, @JsonProperty(TESTRIGS_VAR) SortedSet<String> testrigs) {
-    return new Container(name, testrigs);
+  public static Container makeContainer(
+      @JsonProperty(NAME_VAR) String name, @JsonProperty(TESTRIGS_URI_VAR) String testrigsUrl) {
```

Comment:
The URI is not a property of the container. it's a property of the container resource. we may need to distinguish these things in the object hierarchy.